deploy/efs-csi/03-scc.yaml

```diff
@@ -0,0 +1,37 @@
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  annotations:
+    kubernetes.io/description: 'Highly privileged SCC for the EFS CSI driver DaemonSet.'
+  name: efs-csi-scc
```

Comment:
Is this a custom SCC? Or a clone of an existing one that we are providing for this service account?

Reply:
Well... yes. As noted [here](https://github.com/openshift/managed-cluster-config/pull/287#discussion_r411656007) I started by cloning `privileged` to get it working. It may be possible to distill it further.

Reply:
:+1: